resources/assets/js/components/ProfileClasses.vue

```diff
@@ -430,6 +430,17 @@ export default {
                     baseURL: this.api_url
                 }
             );
+        },
+        openDoorSignWindow: function () {
+            var windowObjectReference;
+            var strWindowFeatures = "menubar=no,toolbar=no,personalbar=no,resizable=yes,scrollbars=yes,status=no,titlebar=no,height=600,width=700,centerscreen=yes";
+
+            windowObjectReference = window.open("/printer-friendly-door-sign", "DoorSign", strWindowFeatures);
```

Comment:
So the redirect will be an issue since we host it a little differently on the server vs when we are developing. This needs the application url. Based on what I've seen on the source you can do `window.open(faculty.env.url+"/printer-friendly-door-sign")`